### PR TITLE
Gsd 2116

### PR DESCRIPTION
**Pull request by aimtrettel**

### CODE REVIEW

File: plugins/omar-oms-plugin/grails-app/controllers/omar/oms/ImageSpaceController.groovy [hunk 1]
```diff
@@ -102,10 +106,10 @@ class ImageSpaceController implements AsyncController
     }
     catch ( e )
     {
-      e.printStackTrace()
+        logger.trace("There was an error in ImageSpaceController line 109", e)
```
Comment (cdowin) [inline plugins/omar-oms-plugin/grails-app/controllers/omar/oms/ImageSpaceController.groovy:109]: The logger will print the class and line number. Is there a more interesting message here?

File: plugins/omar-oms-plugin/grails-app/services/omar/oms/ChipperService.groovy [hunk 2]
```diff
@@ -6,96 +6,114 @@ import javax.imageio.ImageIO
...
-log.info'*' * 50
+    def getTile(ChipperCommand cmd)
+    {
 
-      HashMap chipperOptions = cmd?.toChipperOptions()
-      HashMap result = [:]
-      HashMap chipperResult = [:]
-      String outputFormat = cmd.outputFormat?:"image/png"
-      def hints = [type:outputFormat]
+        log.info '*' * 50
```
Comment (cdowin) [inline plugins/omar-oms-plugin/grails-app/services/omar/oms/ChipperService.groovy:16]: Is "info" the appropriate level here? I don't think we need the stars. Instead, I suggest we prepend the 'cmd' with some unique message that we can grep/search. I.E. "Running getTile with ChipperCommand: <cmd>"

File: plugins/omar-oms-plugin/grails-app/services/omar/oms/ImageSpaceService.groovy [hunk 3]
```diff
@@ -375,8 +381,8 @@ class ImageSpaceService
     }
     catch(e)
     {
-      e.printStackTrace()
-      log.error e.toString()
+//      e.printStackTrace()
```
Comment (cdowin) [inline plugins/omar-oms-plugin/grails-app/services/omar/oms/ImageSpaceService.groovy:384]: Remove comment

File: plugins/omar-oms-plugin/src/main/groovy/omar/oms/ChipperUtil.groovy [hunk 4]
```diff
@@ -220,18 +223,18 @@ class ChipperUtil
...
-    {
-      if(!hints.type.contains("tiff"))
-      {
-        // The only type we will support raw band output is TIFF.
-        // this way we can send back the raw tiff without modification
-        //
-        keepBands = false;
-      }
-    }
+//    Boolean keepBands = hints?.keepBands ->not being used
```
Comment (cdowin) [inline plugins/omar-oms-plugin/src/main/groovy/omar/oms/ChipperUtil.groovy:226]: Remove commented code

File: plugins/omar-oms-plugin/src/main/groovy/omar/oms/JaiImage.groovy [hunk 5]
```diff
@@ -43,7 +43,7 @@ class JaiImage
 
     modifiedImage
   }
-  static def reformatImage(def image, int tileWidth = 256, int tileHeight = 256)
+  static def reformatImage(def image /*, int tileWidth = 256, int tileHeight = 256*/)
```
Comment (cdowin) [inline plugins/omar-oms-plugin/src/main/groovy/omar/oms/JaiImage.groovy:46]: Remove block comment in function "/* */". This makes for a confusing looking function signature.

**Review by cdowin — CHANGES_REQUESTED** [review]

File: plugins/omar-oms-plugin/src/main/groovy/omar/oms/JaiImage.groovy (hunk 5 above)
Comment (McKenzieTugui) [reply plugins/omar-oms-plugin/src/main/groovy/omar/oms/JaiImage.groovy]: Resolved all of the above comments.

**Review by cadowner90 — APPROVED** [review]

**Review by mjkresslein — APPROVED** [review]

**Pull request merged**